feconf.py

```diff
@@ -669,6 +669,7 @@ def get_empty_ratings():
 SKILL_EDITOR_DATA_URL_PREFIX = '/skill_editor_handler/data'
 SKILL_EDITOR_URL_PREFIX = '/skill_editor'
 SKILL_EDITOR_QUESTION_URL = '/skill_editor_question_handler'
+SKILL_MASTERY_DATA_URL = '/skill_mastery'
```

Comment:
Data URLs normally have /data at the end (and a "handler" somewhere in the name).

Reply:
Done.